alvr/xtask/src/build.rs

```diff
@@ -30,7 +30,6 @@ impl Display for Profile {
 pub fn build_server_lib(
```

Comment:
As far as I can tell this went from building a complete lib for SteamVR to kinda useless?

Reply:
This could be used for Monado, if encoders and rendering are provided separately.

Reply:
I'd rather have a static lib in that case, but tbh the way it is currently is also how I've been working with monado in the past, so let's keep it and I can change it if I feel the need